UnitySDK/Assets/ML-Agents/Examples/Pyramids/Scripts/PyramidAgent.cs

```diff
@@ -16,6 +16,7 @@ public class PyramidAgent : Agent
 
     public override void InitializeAgent()
     {
+        Academy.Instance.ConnectToPlayerLoop();
```

Comment:
Doesn't have a Settings GameObject, so call this manually.

Reply:
This does feel a bit weird since it's on an Agent.

Reply:
Moved to PyramidArea.Awake() (and moved it out of other example Agent code too)